src/test/java/org/opentripplanner/graph_builder/linking/LinkStopToPlatformTest.java

```diff
@@ -0,0 +1,100 @@
+package org.opentripplanner.graph_builder.linking;
+
+
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.GeometryFactory;
+import com.vividsolutions.jts.geom.LineString;
+import org.junit.Before;
+import org.junit.Test;
+import org.onebusaway.gtfs.model.AgencyAndId;
+import org.onebusaway.gtfs.model.Stop;
+import org.opentripplanner.common.geometry.GeometryUtils;
+import org.opentripplanner.common.geometry.SphericalDistanceLibrary;
+import org.opentripplanner.openstreetmap.model.OSMWithTags;
+import org.opentripplanner.routing.edgetype.*;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.vertextype.IntersectionVertex;
+import org.opentripplanner.routing.vertextype.TransitStop;
+import org.opentripplanner.util.I18NString;
+import org.opentripplanner.util.LocalizedString;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class LinkStopToPlatformTest {
+
+    private static GeometryFactory geometryFactory = GeometryUtils.getGeometryFactory();
+
+    private Graph _graph;
+
+    @Before
+    public void before() {
+
+        // Set up transit platform
+
+        _graph = new Graph();
+
+        ArrayList<IntersectionVertex> vertices = new ArrayList<IntersectionVertex>();
+
+        vertices.add(new IntersectionVertex(_graph, "1", 10.22054, 59.13568, "Platform vertix 1"));
+        vertices.add(new IntersectionVertex(_graph, "2", 10.22432, 59.13519, "Platform vertix 2"));
+        vertices.add(new IntersectionVertex(_graph, "3", 10.22492, 59.13514, "Platform vertix 3"));
+        vertices.add(new IntersectionVertex(_graph, "4", 10.22493, 59.13518, "Platform vertix 4"));
+        vertices.add(new IntersectionVertex(_graph, "5", 10.22056, 59.13575, "Platform vertix 5"));
```

Comment:
Is this vertex used?

Reply:
They are used when creating the edges just below.